src/gamestarfield.cpp

```diff
@@ -52,12 +56,35 @@ bool GameStarfield::init(IOrganizer* moInfo)
       std::make_shared<StarfieldModDataContent>(m_Organizer->gameFeatures()));
   registerFeature(std::make_shared<GamebryoSaveGameInfo>(this));
   registerFeature(std::make_shared<StarfieldGamePlugins>(moInfo));
-  registerFeature(std::make_shared<StarfieldUnmangedMods>(this));
+  registerFeature(std::make_shared<StarfieldUnmangedMods>(this, localAppFolder()));
   registerFeature(std::make_shared<StarfieldBSAInvalidation>(dataArchives.get(), this));
 
+  m_Organizer->pluginList()->onRefreshed([&]() {
+    setCCCFile();
+  });
+
   return true;
 }
 
+void GameStarfield::setCCCFile() const
+{
+  if (m_Organizer->profilePath().isEmpty())
+    return;
+  if (m_Organizer->pluginSetting(name(), "enable_loadorder_fix").toBool()) {
+    QFile cccFile(m_Organizer->profilePath() + "/Starfield.ccc");
+    if (cccFile.open(QIODevice::WriteOnly)) {
+      ON_BLOCK_EXIT([&cccFile]() {
```

Comment:
I don't think that block is required, the destructor of `QFile` will close the file.

Reply:
Sure. In that case I should remove the example that already existed in the file.